### PR TITLE
Update zh_cn for 1.9+

### PR DESCRIPTION
**Pull request by TartaricAcid**

### CODE REVIEW

**Comment by TehNut** [comment]
This guy does translations for BM, he is credible. 

However, pinging @3TUSK and @qianzha

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+
+gui.antiqueatlas.scalebar=以方块缩放
+gui.antiqueatlas.addMarker=增加标记
+gui.antiqueatlas.delMarker=删除标记
+gui.antiqueatlas.followPlayer=跟随该玩家
+gui.antiqueatlas.exportImage=导出图像
+gui.antiqueatlas.hideMarkers=隐藏标记
+gui.antiqueatlas.showMarkers=显示标记
+
+gui.antiqueatlas.marker.label=标签:
```
Comment (3TUSK) [inline src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: 全角标点，下同

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+gui.antiqueatlas.marker.type=类型:
+gui.antiqueatlas.marker.village=村庄
+gui.antiqueatlas.marker.netherPortal=下界传送门
+gui.antiqueatlas.marker.tomb=%s 在此长眠
+
+gui.antiqueatlas.export.opening=打开文件系统对话框...
+gui.antiqueatlas.export.selectFile=选择PNG格式图片并另存为
+gui.antiqueatlas.export.tooLarge=抱歉!图片过大无法渲染,请尝试增大java堆内存空间.
```
Comment (3TUSK) [inline src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: Are you sure normal user can understand what's "increase java heap size"? I will say "Try allocating more memory to Minecraft" is better, i.e. “请尝试给 Minecraft 分配更多内存空间”。

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang [hunk 3]
```diff
@@ -1,22 +1,36 @@
...
+
+gui.antiqueatlas.export.setup=设置...
```
Comment (3TUSK) [inline src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: `……` （U+2026, HORIZONTAL ELLIPSIS，两个），下同

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang [hunk 4]
```diff
@@ -1,22 +1,36 @@
...
-gui.antiqueatlas.export.opening=打开文件系统对话框...
-gui.antiqueatlas.export.selectFile=选择PNG格式图片并另存为
-gui.antiqueatlas.export.tooLarge=抱歉!图片过大无法渲染,请尝试增大java堆内存空间.
-gui.antiqueatlas.export.rendering=渲染...
-gui.antiqueatlas.export.writing=写入文件...
+item.antiqueAtlas.name=古式地图
+item.emptyAntiqueAtlas.name=空白古式地图
+
+gui.antiqueatlas.scalebar=以方块缩放
+gui.antiqueatlas.addMarker=增加标记
```
Comment (3TUSK) [inline src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: 增加 vs 新建。两个翻译都没有太大问题，选一个好了。

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang [hunk 5]
```diff
@@ -1,22 +1,36 @@
...
+gui.antiqueatlas.export.rendering=渲染...
+gui.antiqueatlas.export.writing=写入文件...
+gui.antiqueatlas.export.loadingtextures=加载材质...
+gui.antiqueatlas.export.makingbuffer=分配 %dx%d 图像缓冲区...
+
+gui.antiqueatlas.export.renderstripe=渲染层 %d/%d...
+gui.antiqueatlas.export.writestripe=写入层...
+
+gui.antiqueatlas.export.rendering.background=渲染地图背景
+gui.antiqueatlas.export.rendering.map=渲染地图标题
```
Comment (3TUSK) [inline src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: tile (as in `TileEntity`) != title. 渲染地图方块？似乎不对劲……

**Review by 3TUSK — CHANGES_REQUESTED** [review]

**Comment by 3TUSK** [comment]
I have been summoned. (doubt that whether qianzha will appear, I remember he (?) is busy recently.)

@liach WayofTime/BloodMagic#1003

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang (hunk 4 above)
Comment (liach) [reply src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: `添加`?

**Review by liach — CHANGES_REQUESTED** [review]
The filename should be cased (`zh_CN.lang`) because this is Minecraft 1.9, not 1.11 yet.

File: src/main/resources/assets/antiqueatlas/lang/zh_cn.lang (hunk 4 above)
Comment (3TUSK) [reply src/main/resources/assets/antiqueatlas/lang/zh_cn.lang]: Possible.

**Comment by 3TUSK** [comment]
@liach [It's a mess](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/tree/1.9.4/src/main/resources/assets/antiqueatlas/lang)

**Comment by liach** [comment]
Merge issues lol

**Comment by TartaricAcid** [comment]
I think the PR can be merge. ^_^

**Review by liach — APPROVED** [review]

**Comment by 3TUSK** [comment]
Second this: rename `zh_cn.lang` back to `zh_CN.lang`?

**Comment by Kenkron** [comment]
So... can I merge it yet?

**Comment by 3TUSK** [comment]
@Kenkron If the file name is correct.

**Review by osipxd — APPROVED** [review]

**Pull request merged**